sdk/devcenter/azure-developer-devcenter/src/samples/README.md

```diff
@@ -0,0 +1,253 @@
+# Introduction
+These files give samples of how to perform simple operations using DevCenter clients. 
+You'll need to create backing resources in order to take full advantage of these scenarios.
+You can find documentation on these resources [here](https://azure.microsoft.com/services/dev-box/).
+
+The minimum requirements to create Dev Box resources using this SDK are to create DevCenter, Project, and Pool resources.
+The minimum requirements to create Environment resources using this SDK are to create DevCenter, Project, EnvironmentType, and CatalogItem resources.
+
+# Dev Box samples
+
+## Setup
+Set the following environment variables for easy consumption in client code:
+
+- `AZURE_TENANT_ID`: GUID identifier for the Azure tenant
+- `DEVCENTER_NAME`: Name of the DevCenter resource
+
+
+## Creating clients
+
+The project is the top-level resource on the data plane. We'll create a dev center client first to access projects:
+```
```

Comment:
In GA version, you should use code snippet tag, similar to ReadmeSamples.java -> README.md

Reply:
Done, thanks!